src/test/Components/ImagesTable/ImagesTable.test.js

```diff
@@ -401,12 +405,19 @@ describe('Images Table', () => {
 
       // render the expected <ImageLink /> and compare the text content for a link
       render(
-        <ImageLink
-          imageStatus={mockStatus[compose.id].image_status}
-          uploadOptions={
-            compose.request.image_requests[0].upload_request.options
-          }
-        />,
+        <BrowserRouter>
+          <ImageLink
+            imageStatus={mockStatus[compose.id].image_status}
+            uploadOptions={
+              compose.request.image_requests[0].upload_request.options
+            }
+            isExpired={
+              mockStatus[compose.id].image_status.status === 'expired'
+                ? true
+                : false
+            }
```

Comment:
This is now checking the expired state based on the mocked image status; not on the delta to created_at.  I'd check based on the delta since that's what we expect here.

Reply:
Updated. In the case of expired images the tests check for a match with a text content.

The tests are now a mix of rendering and text matching. When I've tried to drop the rendering entirely, I've encountered a problem with creating the test output to match against. For a state from the mockupStatus to be mapped correctly, it would need to go through `setComposeStatus()` function in `ImagesTable.js` first and after that through `ImageBuildStatus.js`